integration/convert_test.go

```diff
@@ -198,6 +198,28 @@ func validateConversion(t *testing.T, sh *shell.Shell, originalDigest, converted
 			t.Errorf("manifest %v does not contain expected soci index digest %v", manifestDesc, sociIndexDesc.Digest)
 		}
 	}
+
+	// Get GC labels
+	convertedLabelsOutput := sh.O("ctr", "content", "label", convertedDigest)
+	originalLabelsOutput := sh.O("ctr", "content", "label", originalDigest)
+
+	// Parse labels
+	convertedLabels := strings.Split(strings.TrimSpace(string(convertedLabelsOutput)), ",")
+	originalLabels := strings.Split(strings.TrimSpace(string(originalLabelsOutput)), ",")
+
+	if len(convertedLabelsOutput) == 0 {
+		convertedLabels = []string{}
+	}
+	if len(originalLabelsOutput) == 0 {
+		originalLabels = []string{}
+	}
+
+	// Validate that conversion added GC labels
+	if len(convertedLabels) < len(originalLabels) {
```

Comment:
This can fail. 
For example, in alpine image:

Original GC labels:
```
containerd.io/distribution.source.docker.io=library/alpine,
containerd.io/gc.ref.content.config=sha256:7acffee03fe864cd6b88219a1028855d6c912e7cf6fac633aa4307529fd0cc08,
containerd.io/gc.ref.content.l.0=sha256:014e56e613968f73cce0858124ca5fbc601d7888099969a4eea69f31dcd71a53
```

Converted GC labels:
```
containerd.io/gc.ref.content.config=sha256:7acffee03fe864cd6b88219a1028855d6c912e7cf6fac633aa4307529fd0cc08,
containerd.io/gc.ref.content.l.0=sha256:014e56e613968f73cce0858124ca5fbc601d7888099969a4eea69f31dcd71a53
```